docs/ApiMenu.json

```diff
@@ -0,0 +1,30 @@
+{ "data": [
```

Comment:
Move the "data" key to the second line and adjust indentation to reflect that change.